packages/cli/README.md

```diff
@@ -20,15 +20,16 @@ This is the Command Line Interface (CLI) package for creating designs using [Nod
 - [Documentation](#documentation)
 - [Contributing](#contributing)
 - [Community](#community)
-- [Acknowledgements](#acknowledgements)
+- [Backers](#backers)
+- [Sponsors](#sponsors)
 - [License](#license)
 
 ## Usage
 
 ### Install Node.js
 
 > IMPORTANT: You need a recent, LTS version of [Node.js](http://nodejs.org/)
-> Click on build "passing" above to understand which versions of Node.js are supported. Other versions of Node.js are not guaranteed to work !
+> Other versions of Node.js are not guaranteed to work!
```

Comment:
Hmm Sorry to be a pain, but this provides a lot less information ?
Perhaps we can just link to the versions, or say clearly which versions are supported, but this is less informative as it is

Reply:
😃 I added that before, and removed it. Ok. I’ll add that back.

Reply:
I added this back.